boba_community/fraud-detector/docker-detect.yml

```diff
@@ -0,0 +1,105 @@
+version: "3.4"
+
+x-var: &L1_NODE_WEB3_URL
+  L1_NODE_WEB3_URL=https://mainnet.infura.io/v3/INFURA_KEY
+
+x-var: &DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT
+  DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT=https://mainnet.infura.io/v3/INFURA_KEY
+
+x-var: &L2_NODE_WEB3_URL
+  L2_NODE_WEB3_URL=http://geth_l2:8545
+
+x-var: &DEPLOYER_HTTP
+  DEPLOYER_HTTP=http://deployer:8081
+
+x-var: &DEPLOYER_URL
+  DEPLOYER_URL=http://deployer:8081
+
+x-var: &ADDRESS_MANAGER_ADDRESS
+  ADDRESS_MANAGER_ADDRESS=0x8376ac6C3f73a25Dd994E0b0669ca7ee0C02F089
+
+x-var: &DATA_TRANSPORT_LAYER__ADDRESS_MANAGER
+  DATA_TRANSPORT_LAYER__ADDRESS_MANAGER=0x8376ac6C3f73a25Dd994E0b0669ca7ee0C02F089
+
+x-var: &ETH1_CTC_DEPLOYMENT_HEIGHT
+  ETH1_CTC_DEPLOYMENT_HEIGHT=13011896
+
+services:
+
+  deployer:
```

Comment:
Please remove the deployer. I have open the port of the replica. The address of `addresses.json` is `https://replica.boba.network:8081/addresses.json` and the address of the `state-dump.latest.json` is `https://replica.boba.network:8081/state-dump.latest.json`.

Reply:
Done